cleanup_utils/integration_cleanup.py

```diff
@@ -109,6 +109,8 @@ def run_integration_cleanup(dry_run=False):
         dry_run=dry_run
     )
     mysql_cleanup.run_cleanup()
+    if os.environ.get('DROP_INTEGRATION_DBS_AND_USERS', 'False').lower() == 'true':
```

Comment:
Should we set this variable on CircleCI Environment?

Reply:
The idea is to use it every once in a while when we want to delete every database and user used in the integration tests.
For the very day runs we probably don't want to delete everything or tests running in parallel will not work properly.
If things are getting a bit slow we could just have a user with admin access to set this variable in CirlceCI and then run the cleanup step.

Reply:
Seems ok to leave this here.